docs/reference/interpreting.md

```diff
@@ -367,3 +370,109 @@ The following interpretation would create a `Locality` node with the keys of `Ne
             data: !jmespath city
           state: !jmespath state
 ```
+
+### Source Node and Relationship Interpretation Combined
```

Comment:
Can we move this section after `Switch` so that it is not buried in the docs for the Value Providers? 
<img width="451" alt="Screenshot 2024-04-18 at 7 46 51 AM" src="https://github.com/nodestream-proj/docs/assets/22159145/031d4e1c-de62-421d-b659-87d9128568d1">